code/modules/fishing/fishing_spots.dm

```diff
@@ -26,15 +26,42 @@ ABSTRACT_TYPE(/datum/fishing_spot)
 	/// the type of the atom that is the "fishing spot"
 	var/fishing_atom_type = null
 	/// associative list with the format (fish_type = probability), doesnt need to be ordered in descending probability
+	/// these are the fishing results that are ALWAYS avaiable at the spot. These won't get modified by conditionals like fishing loottables are.
 	var/list/fish_available = null
```

Comment:
This is bound to cause problems, it's not really obvious from the variable name that these are innate and unaffected by loot tables. Comment is nice, but I'd prefer a more descriptive var name that makes it hyper-obvious.

Might almost consider doing away with this one entirely and replacing every instance of "fish_available" with a lootpool corresponding to the location, but that can be done in a follow-up PR.